.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
You can delete this

Reply:
This came through vs code. WIll delete this.